src/main/java/org/jabref/logic/msbib/MSBibMapping.java

```diff
@@ -120,47 +128,36 @@ public class MSBibMapping {
         BIBLATEX_TO_MS_BIB.put(new UnknownField(MSBIB_PREFIX + "productioncompany"), "ProductionCompany");
     }
 
+    static {
+        BIB_ENTRYTYPE_MAPPING.put(StandardEntryType.Book, MSBibEntryType.Book);
```

Comment:
Inconsistent usage, for the others you use Map.of...

Reply:
Map.of does not support that much parameters. I tried ,and. Checked jdk source